MyLibrary/Tests/ScheduleFeatureTests/ScheduleTests.swift

```diff
@@ -13,12 +14,14 @@ final class ScheduleTests: XCTestCase {
       $0[DataClient.self].fetchDay1 = { @Sendable in .mock1 }
       $0[DataClient.self].fetchDay2 = { @Sendable in .mock2 }
       $0[DataClient.self].fetchWorkshop = { @Sendable in .mock3 }
+      $0[FileClient.self].loadFavorites = { @Sendable in .mock1 }
```

Comment:
Should be separated your test.

Reply:
I'm sorry that I can't image way to separate.

Current implementation, loading favorites using `FileClient`  and fetching conferences using `DataClient` at the same time (`onAppear`).
So it's required to implement `testValue` of `loadFavorites` to test `onAppear` behavior.

I could only image to separate `loadResponse(Result<Favorites, Error>)` from `fetchResponse(Result<SchedulesResponse, Error>)`, but this idea also requires `testValue` of `loadFavorites`, so I think it doesn't satisfy your review.

Reply:
Since fetching data using `DataClient` and loading favorites using `FileClient` are must to be executed at the same time, onAppear, I renamed `testFetchData` to `testOnAppear` in 04cefc07faeec895fb6e6ec5cb9806d040e55806.